### PR TITLE
Target headers

### PR DESCRIPTION
**Pull request by ag12**
WIP.

### CODE REVIEW

File: FinniversKit.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1428,7 +1435,8 @@
 				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
 				PRODUCT_BUNDLE_IDENTIFIER = no.finn.FinniversKit;
-				PRODUCT_NAME = "$(TARGET_NAME:c99extidentifier)";
```
Comment (ag12) [inline FinniversKit.xcodeproj/project.pbxproj]: What is this?

File: FinniversKit.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -121,6 +121,9 @@
 		AA7313D5202078D4008A7251 /* RegisterViewModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = AA7313D4202078D4008A7251 /* RegisterViewModel.swift */; };
```
Comment (ag12) [inline FinniversKit.xcodeproj/project.pbxproj]: Cant find broadcast * Views.

**Comment by ag12** [comment]
https://stackoverflow.com/questions/24070722/swift-how-can-i-create-external-interface-for-static-library-public-headers-an

File: FinniversKit.xcodeproj/project.pbxproj (hunk 1 above)
Comment (3lvis) [reply FinniversKit.xcodeproj/project.pbxproj]: I think this is required for Carthage to build a .framework

Comment (ag12) [reply FinniversKit.xcodeproj/project.pbxproj]: Ok, I will delete this PR/ branch. We can have a discussion about it later :)

**Pull request closed without merging**